statsmodels/tools/eval_measures.py

```diff
@@ -67,6 +68,30 @@ def rmse(x1, x2, axis=0):
     x2 = np.asanyarray(x2)
     return np.sqrt(mse(x1, x2, axis=axis))
 
+def rmspe(y, y_hat, axis=0, zeros=np.nan):
+    """
+    root mean squared percentange error
+    
+    Parameters
+    ----------
+    y, y_hat : array_like
+       The performance measure depends on the difference between these two
+       arrays.
+    axis : int
+       axis along which the summary statistic is calculated
+    zeros : float
+       value to assign to error where actual value is zero
+
+    Returns
+    -------
+    rmse : ndarray or float
+       root mean squared error along given axis.
+    """
+    y_hat = np.asanyarray(y_hat)
```

Comment:
Either asarray (no subclasses please) or use array_like from statsmodels.tools.validation.

Reply:
And again, I'm just aligning with the rest of the script.

There may be some obsolete conventions used in the script, but my intention here is simply to add a new feature.

Reply:
The old code may be buggy in some edge cases.  We cannot accept new code that maintains the same bad behavior. 

I'm not sure why you are putting your foot down for small changes. If you make these then this can be easily merged.

Reply:
Haha, sorry. I can change it :)

Reply:
Done now